plugin-default-impl/src/main/java/com/alibaba/nacos/plugin/auth/impl/persistence/EmbeddedUserPersistServiceImpl.java

```diff
@@ -111,17 +111,24 @@ public Page<User> getUsers(int pageNo, int pageSize, String username) {
         PaginationHelper<User> helper = createPaginationHelper();
         
         String sqlCountRows = "SELECT count(*) FROM users ";
-        
+
```

Comment:
Please use nacos code style to reformat codes.

Reply:
I'm sorry, I have made the changes.